langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/statements/foreach/ForeachNegativeTests.java

```diff
@@ -38,37 +38,37 @@ public void testForeachSemanticsNegative() {
         int index = 0;
         BAssertUtil.validateError(compile, index++,
                 "invalid list binding pattern: attempted to infer a list type, but found 'string'",
-                39, 17);
+                39, 13);
```

Comment:
Old positions of these error messages seem to be correct. Now the error is given to the whole `typed-binding pattern` instead of `list-binding-pattern`. Should we correct it from where the error is logged?

Reply:
Yeah old positions seams to be correct